docs/kf.dev/content/en/docs/operators/customizing-buildpacks/_index.md

```diff
@@ -0,0 +1,201 @@
+        Spec: v1alpha1.SpaceSpec{
+                Security: v1alpha1.SpaceSpecSecurity{EnableDeveloperLogsAccess: true},
+                BuildpackBuild: v1alpha1.SpaceSpecBuildpackBuild{
+-                       BuilderImage:      "gcr.io/kf-releases/buildpack-builder:latest",
++                       BuilderImage:      "heroku/buildpacks",
+                        ContainerRegistry: "gcr.io/kf-install-instructions",
+                        Env:               nil,
+                },
+                Execution:      v1alpha1.SpaceSpecExecution{Domains: []v1alpha1.SpaceDomain{{Domain: "evanbrown.io", Default: true}}},
+                ResourceLimits: v1alpha1.SpaceSpecResourceLimits{},
+        },
+        Status: v1alpha1.SpaceStatus{Status: v1beta1.Status{Conditions: v1beta1.Conditions{{Type: "AuditorRoleReady", Status: "True", LastTransitionTime: apis.VolatileTime{Inner: v1.Time{Time: s"2019-08-20 10:00:07 -0700 PDT"}}}, {Type: "DeveloperRoleReady", Status: "True", LastTransitionTime: apis.VolatileTime{Inner: v1.Time{Time: s"2019-08-20 10:00:07 -0700 PDT"}}}, {Type: "LimitRangeReady", Status: "True", LastTransitionTime: apis.VolatileTime{Inner: v1.Time{Time: s"2019-08-20 10:00:07 -0700 PDT"}}}, {Type: "NamespaceReady", Status: "True", LastTransitionTime: apis.VolatileTime{Inner: v1.Time{Time: s"2019-08-20 10:00:07 -0700 PDT"}}}, {Type: "Ready", Status: "True", LastTransitionTime: apis.VolatileTime{Inner: v1.Time{Time: s"2019-08-20 10:00:07 -0700 PDT"}}}, {Type: "ResourceQuotaReady", Status: "True", LastTransitionTime: apis.VolatileTime{Inner: v1.Time{Time: s"2019-08-20 10:00:07 -0700 PDT"}}}}}},
+  }
+```
+
+This will output a "Space diff", highlighting what changed in the space. You can view the buildpacks made available to your space by the `heroku/buildpacks` image by running `kf buildpacks`:
+
+```sh
+$ kf buildpacks
+
+ng buildpacks in space: buildpack-docs
+NAME            POSITION VERSION    LATEST
+heroku/java     0        0.13       true
+heroku/ruby     1        0.0.1      true
+heroku/procfile 2        0.2        true
+heroku/python   3        v0.0.5.0.1 true
+heroku/gradle   4        v0.0.5.0.1 true
+heroku/php      5        v0.0.5.0.1 true
+heroku/go       6        v0.0.5.0.1 true
+heroku/nodejs   7        v0.0.5.0.1 true
+```
+
+### Create your own builder image
+[create-builder]: https://buildpacks.io/docs/using-pack/working-with-builders/
+The [Buildpack CLI `pack`][pack] is used to create your own builder image. You can follow `pack`'s [Working with builders using `create-builder`][create-builder] documentation to create your own builder image. Once created, push it to a Docker registry and use the `kf config-space` command to configure your Kf space to use the new builder image:
```

Comment:
nit: s/Docker/container

Reply:
I need to `s/Docker/container/g` my brain. Fixed.